src/assets/stylesheets/invite-dialog.scss

```diff
@@ -119,6 +119,15 @@
   }
 }
 
+:local(.linkcode){
```

Comment:
Please adjust this to use the `link-code` naming convention, and then use `linkCode` to reference the styles in the JS file. thanks!

Reply:
Yes, i'm sorry i almost forgot the naming conventions. But i've fixed the naming conventions ! Hope this works :)

Reply:
And i'd love to contribute more do tag me in issues if you want me to :)